presto-orc/src/main/java/com/facebook/presto/orc/Filter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+public abstract class Filter
```

Comment:
Maybe `TupleDomainFilter`? That could merge `Filters` and `TupleDomainFilters` utils

Reply:
Could we briefly explain in a comment or javadoc what these testXXX functions are doing? Are they to test if the given value is in the range of the filter? Maybe @wenleix can provide naming suggestions.